src/pudl/settings.py

```diff
@@ -618,103 +617,6 @@ class EtlSettings(BaseSettings):
     pudl_in: str = pudl.workspace.setup.get_defaults()["pudl_in"]
     pudl_out: str = pudl.workspace.setup.get_defaults()["pudl_out"]
 
-    @root_validator(pre=False)
```

Comment:
I removed this because I don't think it this is relevant anymore now that all tables are being loaded.